snap/local/start-mongod.sh

```diff
@@ -2,4 +2,4 @@
 
 # For security measures, daemons should not be run as sudo. Execute mongod as the non-sudo user: snap-daemon.
 exec $SNAP/usr/bin/setpriv --clear-groups --reuid snap_daemon \
-  --regid snap_daemon -- $SNAP/usr/bin/mongod --config ${SNAP_COMMON}/mongod.conf "$@"
+  --regid snap_daemon -- $SNAP/usr/bin/mongod --config ${SNAP_COMMON}/mongod.conf ${MONGOD_ARGS} "$@"
```

Comment:
looks good! One point is that I think modifications are needed to ensure that `${SNAP_COMMON}/mongod.conf` is empty :)

Reply:
Why do you want to ensure that its empty?  Isn't the idea that the user could provide a config as well?

Reply:
I apologise for the private link in an opensource git but [I think the consensus was to keep the file empty](https://chat.canonical.com/canonical/pl/xbe6j1fz8bn9tkoahe5atefufh) :)

Reply:
Also sorry, I realise the confusion, I am bad at explaining 🤦‍♀️ - would you mind adding extra code that has the `mongo.conf` start as an empty file

Reply:
The problem with that is I think it makes sense for the snap to be functional outside of the charm.  Providing a default config file is required for that to be the case.  Why not wipe the config file in the charm so that the snap works without having to provide a config?

Reply:
Hmm that sounds reasonable to me, good idea. Is that okay with you @delgod ?

Reply:
Checked with Mykola this is okay :)